From 6a85f1fb86728f919f534fe6af57c6a212f65196 Mon Sep 17 00:00:00 2001
From: Julien Olivain <julien.olivain@nxp.com>
Date: Wed, 16 Nov 2016 09:20:25 +0100
Subject: [PATCH 2217/5242] MLK-13472: hwmon: mxc_mma8451: add empty sentinel
 entry at the end of i2c_device_id table

commit  9fe156817e58a70a7e8cf45665d5beb30b13c637 from
https://source.codeaurora.org/external/imx/linux-imx.git

This is fixing the build when the driver is enabled as a module, when
CONFIG_MXC_MMA8451=m

Signed-off-by: Julien Olivain <julien.olivain@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/hwmon/mxc_mma8451.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/hwmon/mxc_mma8451.c b/drivers/hwmon/mxc_mma8451.c
index f20fb8e..9d6c539 100644
--- a/drivers/hwmon/mxc_mma8451.c
+++ b/drivers/hwmon/mxc_mma8451.c
@@ -556,6 +556,7 @@ static int mma8451_resume(struct device *dev)
 
 static const struct i2c_device_id mma8451_id[] = {
 	{"mma8451", 0},
+	{ /* sentinel */ }
 };
 
 MODULE_DEVICE_TABLE(i2c, mma8451_id);
-- 
1.7.9.5

